CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+This project adheres to [Semantic Versioning](http://semver.org/).
```

Comment:
This project can not adhere to semantic versioning as there is only one published product.

Reply:
Why cant it?

Reply:
Users cannot pick a version, hence semantic versioning is redundant. They have have to use the latest version.

Reply:
Same argument could be made for the rest of the Productivity Suite...